karapace/karapace.py

```diff
@@ -157,6 +157,7 @@ def _create_producer(self):
             ssl_certfile=self.config["ssl_certfile"],
             ssl_keyfile=self.config["ssl_keyfile"],
             api_version=(1, 0, 0),
+            metadata_max_age_ms=60000,
```

Comment:
Could we take these out of the config file and default to 60k?

Reply:
Good idea :+1: